### PR TITLE
Add methods to add and remove repos for actions on org level

### LINKED ISSUE #1995: Github Actions SetEnabledRepos functionality is missing
Currently we can list enabled repos on an org for github actions:
https://github.com/google/go-github/blob/b1b5e4e9d0ebffbb88e5c19958c87462fb94f169/github/actions_runners.go#L243-L265

However i dont see the Put and Delete method implemented yet for https://docs.github.com/en/rest/reference/actions#set-selected-repositories-enabled-for-github-actions-in-an-organization

### PR DESCRIPTION
**Pull request by FloThinksPi**
This commit adds the abillity to add one or more repositories to the
list of enabled repositories for github actions on org level.
It also adds the abillity to remove them from this list.
With this change we can now enable and disable repos to use github actions on org level.

The used endpoints for this are:
https://docs.github.com/en/rest/reference/actions#set-selected-repositories-enabled-for-github-actions-in-an-organization
https://docs.github.com/en/rest/reference/actions#disable-a-selected-repository-for-github-actions-in-an-organization

fixes #1995

### CODE REVIEW

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:288]:
```suggestion
// RemoveEnabledRepoInOrg removes a single repository from the list of enabled repos for GitHub Actions in an organization.
```

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:269]:
```suggestion
// GitHub API docs: https://docs.github.com/en/rest/reference/actions#set-selected-repositories-enabled-for-github-actions-in-an-organization
```

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:274]:
```suggestion
		IDs []int64 `json:"selected_repository_ids"`
```

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:275]:
```suggestion
	}{IDs: repositoryIDs})
```

File: github/actions_runners_test.go
Comment (gmlewis) [inline github/actions_runners_test.go:431]: Please delete this blank line for consistency with the rest of the repo.

File: github/actions_runners_test.go
Comment (gmlewis) [inline github/actions_runners_test.go:459]: ditto

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thanks, @FloThinksPi , but this PR appears to have a few problems.
Did you try out these two new methods before submitting the PR?

I ask because one of the signatures doesn't match the official documentation and I'm not sure how it could have worked.

**Comment by FloThinksPi** [comment]
Woh thanks @gmlewis  for the super quick review. 
I just wanted to display the diff and must have opened it already. Didnt run it yet at all, just the tests. 
I will include your suggestions and try it out.

**Comment by FloThinksPi** [comment]
@googlebot I signed it!

**Comment by gmlewis** [comment]
Hi @FloThinksPi - it looks like the tests need updating:

```
=== RUN   TestActionsService_AddEnabledReposInOrg
    actions_runners_test.go:411: request Body is {"selected_repository_ids":[123,1234]}
        , want {"repository_id":[123,1234]}
--- FAIL: TestActionsService_AddEnabledReposInOrg (0.00s)
```

**Comment by gmlewis** [comment]
@googlebot I consent.

**Comment by FloThinksPi** [comment]
@gmlewis Yea all done now 👍🏼 . I also renamed the AddEnableReposInOrg method to SetEnabledReposInOrg since the used endpoint sets the list absolute. I thus added the https://docs.github.com/en/rest/reference/actions#enable-a-selected-repository-for-github-actions-in-an-organization endpoint which is now AddEnableReposInOrg which appends to the list of enabled repos.

All tests return ok and i tested all three methods against a GitHubEnterprise 3.0 Instance.

**Comment by gmlewis** [comment]
> Should i squash the commits ?

You don't need to squash, as we only "Squash and merge" in this repo anyway.

I'll check the names now.

**Review by gmlewis — APPROVED** [review]
Thank you, @FloThinksPi !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
